source/ADIOS.cpp

```diff
@@ -25,10 +25,8 @@
 #include "engine/bp/BPFileReader.h"
 #include "engine/bp/BPFileWriter.h"
 
-#ifdef ADIOS_HAVE_DATAMAN // external dependencies
 #include "engine/dataman/DataManReader.h"
```

Comment:
Still though, if built 'fully static' I don't think it will work.  So rather than condition it's inclusion in whether shared libs are being used, it should still be conditioned on whether shared libs are even supported by the platform.  See in the top level CMakeLists.txt how ADIOS_USE_PIC is set, it uses the same check.